controllers/suite_test.go

```diff
@@ -164,10 +149,12 @@ var _ = BeforeSuite(func(specCtx SpecContext) {
 
 var _ = AfterSuite(func() {
 	cancel()
-	ias.ReadCredentials = originalReadCredentialsFunc
+	revertSkrNewClientStub()
+	revertReadCredentialsStub()
 	if !existIasCreds() {
-		ias.NewIasClient = originalNewIasClientFunc
+		revertIasNewClientStub()
 	}
+	targetClusterTestEnv.ControlPlane.GetAPIServer()
```

Comment:
nitpick
```suggestion
	skrClusterTestEnv.ControlPlane.GetAPIServer()
```

Reply:
This was there accidentally and is already removed. 😄